Jobs.Fetcher.Facebook/Client/Fetcher.cs

```diff
@@ -278,34 +291,66 @@ public class Fetcher {
             /**
                Fetch children on the `edge` api api path of the social media artifact `node`.
              */
+            if (!node.TryGetValue("id", out var id)) {
+                Logger.Warning($"Could get ID from node of {edge.Name}");
+                return;
+            }
             JObject tip;
             try {
-                tip = FetchEndpoint(edge, node["id"].ToString(), edge.Name, edge.Columns.Select(x => x.Value).ToList(), edge.Ordering).Result;
-            } catch (FacebookApiException) {
+                tip = FetchEndpoint(edge, id.ToString(), edge.Name, edge.Columns.Select(x => x.Value).ToList(), edge.Ordering).Result;
+            } catch (FacebookApiException e) {
+                Logger.Warning($"Couldn't fetch data from edge {edge.Name} due to API exception.");
+                Logger.Debug($"Error: ({e}).");
+                return;
+            } catch (Exception e) {
+                Logger.Warning($"Couldn't fetch data from edge {edge.Name}.");
+                Logger.Debug($"Error: ({e}).");
```

Comment:
I think I would log this (and other unknown exceptions) with a greater level of severity, since it is something we should probably come back to fix